tests/test_capnproto.py

```diff
@@ -462,6 +470,13 @@
     bytes.fromhex("E93601" + "0B" + "8BFF04030201"),  # ELT_PROPERTY_STATUS
 ]
 
+xfailing = [
+    (
+        bytes.fromhex("52" + "09" + "9040" + "a244" + "ff0000"),
```

Comment:
At first sight, do we know why this is failing?

Reply:
No, as I didn't want to delve into construct-specific code for now.